t/37-limit_assets.t

```diff
@@ -93,18 +92,15 @@ sub prepare_asset_status {
             next;
         }
 
-        # ignore 'Core-7.2.iso' and other assets which may or may not exist
-        #  (eg. 'Core-7.2.iso' is downloaded conditionally in 14-grutasks.t)
-        my %ignored_assets = (
-            'iso/Core-7.2.iso'              => 1,
-            'iso/whatever.iso'              => 1,
-            'hdd/hdd_image2.qcow'           => 1,
-            'hdd/hdd_image2.qcow2'          => 1,
-            'hdd/00099963-hdd_image3.qcow2' => 1,
+        # other tests may be clobbering the assets folder depending on order of execution
```

Comment:
So why remove Core-7.2.iso from it?

Reply:
The test was ignoring `Core-7.2.iso` before. I switched it to a whitelist which naturally won't contain that file.

Reply:
Having a well defined list of assets the test considers is actually a good idea.